toolkit/modules/resync/data_models/nodes/stavanger:ShopScenario.node.yaml

```diff
@@ -0,0 +1,506 @@
+- externalId: shop_scenario_stavanger_base
+  sources:
+  - properties:
+      commands:
+        externalId: shop_commands_default
+        space: '{{power_ops_instance_space}}'
+      model:
+        externalId: shop_model_stavanger
+        space: '{{power_ops_instance_space}}'
+      name: Stavanger base
+      source: resync
+      timeResolution:
+        externalId: shop_time_resolution_default
+        space: '{{power_ops_instance_space}}'
+    source:
+      externalId: ShopScenario
+      space: '{{power_ops_models_space}}'
+      type: view
+      version: '{{power_ops_data_model_version}}'
+  space: '{{power_ops_instance_space}}'
+  type:
+    externalId: ShopScenario
+    space: '{{power_ops_type_space}}'
+- externalId: shop_scenario_stavanger_base_15_min_test
+  sources:
+  - properties:
+      commands:
+        externalId: shop_commands_default
+        space: '{{power_ops_instance_space}}'
+      model:
+        externalId: shop_model_stavanger
+        space: '{{power_ops_instance_space}}'
+      name: Stavanger base 15 min test
+      source: resync
+      timeResolution:
+        externalId: 15min_1h_4h:0_24_72
+        space: '{{power_ops_instance_space}}'
+    source:
```

Comment:
Unsure where this got generated (i assume it was generated) but I changed it from this strange format to match the shop commands pattern, so it should become `shop_time_resolution_default`